pylexirumah/plot_parameters.py

```diff
@@ -1,79 +1,24 @@
+# coding: utf8
 """Plot the number of filled-in parameters for each language.
 
 parameter_sampled: map languages to sets of parameters
 """
-
-# CLI libraries
 import sys
 import argparse
+import colorsys
+import re
+import json
+from collections import defaultdict, Counter
 
-# Plotting libraries
 import numpy
-import colorsys
```

Comment:
I tried to install colorsys, and was somewhat confused until I realized it's in the standard lib :)